Training_Center_Task_10/Tests.cs

```diff
@@ -0,0 +1,317 @@
+            IWebElement ClickMeButton = _driver.FindElement(By.CssSelector("button[onclick='myConfirmFunction()']"));
+            ClickMeButton.Click();
+
+            //Pick alert text
+            string alertText = _driver.SwitchTo().Alert().Text;
+
+            Assert.AreEqual("Press a button!", alertText);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptConfirmBoxVerifyAlertClickedOK()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click me!" button
+            IWebElement ClickMeButton = _driver.FindElement(By.CssSelector("button[onclick='myConfirmFunction()']"));
+            ClickMeButton.Click();
+
+            //Cliok OK on alert
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement alertClickedOk = _driver.FindElement(By.Id("confirm-demo"));
+
+            Assert.AreEqual("You pressed OK!", alertClickedOk.Text);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptAlertBoxNameEnteredCorrectly()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click for Prompt Box" button
+            IWebElement ClickForPromptBoxButton = _driver.FindElement(By.CssSelector("button[onclick='myPromptFunction()']"));
+            ClickForPromptBoxButton.Click();
+
+            //Enter the name in alert and click OK
+            _driver.SwitchTo().Alert().SendKeys("Ivo");
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement message = _driver.FindElement(By.Id("prompt-demo"));
+
+            Assert.AreEqual("You have entered 'Ivo' !", message.Text);
+        }
+
+        [Description("Task 7")]
+        [TestMethod]
+        public void WaitForUser()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/dynamic-data-loading-demo.html");
+
+            //Click "Get New User" button and get photo URL of first user
+            IWebElement getNewUserButton = _driver.FindElement(By.Id("save"));
+            getNewUserButton.Click();
+
+            IWebElement userPictureFirstUser = _driver.FindElement(By.CssSelector("img[src^='https://randomuser.me/api/portraits/']"));
+            string urlFirstUser = userPictureFirstUser.GetAttribute("src");
+
+            //Click Get New User button to load a new user
+            getNewUserButton.Click();
+
+            //Verify "loading..." message is displayed
+            IWebElement loadingMessage = _driver.FindElement(By.Id("loading"));
+            Assert.AreEqual(true, loadingMessage.Displayed);
+
+            //Verify that after waiting a new user is displayed. Compare the URLs of first and second user pictures - they should be different.
+            bool getNserButtonDisplayed = getNewUserButton.Displayed;
```

Comment:
Actual Image and F Name and L Name are not displayed yet. Juust loading on the screen

Reply:
My misunderstanding here. Fixed logic so it should verify whether loading message is shown and a new user appears.
Note: also added a logic to check whether user 1 is the same as user 2. But sometimes user 1 and 2 are the same person, so the test fails. I left it commented for now. If it is necessary to add it, say so and I will do it. I simply do not understand whether the task is to check whether a new user is shown, or simply a user is shown after you click the Get new user button.